6103.fix.md

```diff
@@ -0,0 +1,2 @@
+Absolute dependencies (dependenies on tasks in a specified cycle rather than at a specified offset) are now visible in the GUI beyond the specified cycle.
```

Comment:
```suggestion
Absolute dependencies (dependencies on tasks in a specified cycle rather than at a specified offset) are now visible in the GUI beyond the specified cycle.
```